custom-namespaces.yaml

```
@@ -9,6 +9,16 @@ operational_monitoring:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.operational_monitoring_derived.projects_v1
+combined_browser_metrics:
+  glean_app: false
+  owners:
+    - gkaberere@mozilla.com
+  pretty_name: Combined Browser Metrics
+  views:
+    projects:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.telemetry.active_users_aggregates_v1
```

Don't see a version for the view [here](https://github.com/mozilla/bigquery-etl/blob/20bd0301e850c89941f076338bd07a2a7dabab22/sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql#L2) 

```suggestion
        - table: moz-fx-data-shared-prod.telemetry.active_users_aggregates
```

---

will fix, thanks